ui/webui/resources/br_elements/br_toolbar/br_toolbar.js

```diff
@@ -0,0 +1,177 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
what are all these files?

Reply:
@bridiver Could you clarify the question? This specific file, br-toolbar.js, is a module that renders a navigation bar on top of many of the webui pages. It goes with everything else in the br_toolbar dir. Its public API exactly matches the cr-toolbar component so that it can be a drop-in replacement.
It's best to review this WIP via the separate commits as the majority of the files are font files.